airbyte-integrations/connectors/source-dynamodb/source_dynamodb/constants.py

```diff
@@ -0,0 +1,6 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+MAX_ITEMS_PAGINATOR: int = 50
+MAX_PAGES_PAGINATOR: int = 10
```

Comment:
You're using this only in one file I think it's better move to the file to improve readability.